### ISSUE TITLE
Check wording and if the docket number question should mandatory

### ISSUE THREAD

**Issue by luke-karis** [SuffolkLITLab/docassemble-PetitionToChangeNameOfAdult#24]

**Comment by anna-stone** [comment]
I don't think we need to ask for the docket number at all. Typically, the docket number is something the court assigns to a case once something is filed. In the case of a name change petition, the user would not know the docket number for their petition until the petition was submitted to the court and the clerk assigned a number.
